documentation/api-key-format.md

```diff
@@ -1,3 +1,6 @@
+
+[<img src=/images/WasThisHelpful.png width="200"/>](https://www.research.net/r/DGDQWXH?src=api-key-format)
```

Comment:
```suggestion
[<img src=/images/WasThisHelpful.png width="200"/>](https://www.research.net/r/DGDQWXH?src=%2Fdocumentation%2Fapi-key-format.md)
```

Is it possible to get the full subpath of the document so we can avoid collisions and be able to differentiate, for example, which README file they are leaving feedback on?

Reply:
I should be able to do that - I currently strip away the path in favor of just keeping the file name so that the parameter wouldn't look so unwieldy, but then we do have a couple collisions.